src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj

```diff
@@ -66,6 +66,7 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.LanguageServer.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.LanguageServer.Protocol" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.LanguageServer.Protocol.UnitTests" />
+    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.LanguageServer.Xaml" />
```

Comment:
this seems bad.  we shoudl be using EA.  exposing all workspace internals to Xaml is not something i feel comfortable doing with.  we could basically change anything at any time and break you.

Reply:
This is for the new XAML LanguageServer assembly in Roslyn. I did add an EA assembly for our external implementations, but that reminds me that I need to adjust some of the names there.

Reply:
why does the new xaml language server get IVT to roslyn workspace.  again, that seems bad to me :-/

Reply:
Biggest issue is IGlobalOptionService which is used for formatting QuickInfo and CompletionItem tooltips. There are various other "LanguageServices" a few lines further down that also have IVT, including the other XAML one.